src/components/tab-title/tab-title.tsx

```diff
@@ -232,7 +300,6 @@ export class TabTitle implements InteractiveComponent {
     if (this.disabled) {
       return;
     }
-
```

Comment:
For readability, let's restore this newline (the return and emitting have more visual separation with the newline).

Applies to similar changes.